tool/client.cc

```diff
@@ -474,7 +478,7 @@ static int verify_cb(int ok, X509_STORE_CTX *ctx)
   X509 *err_cert = X509_STORE_CTX_get_current_cert(ctx);
   int err = X509_STORE_CTX_get_error(ctx);
   int depth = X509_STORE_CTX_get_error_depth(ctx);
-  bssl::UniquePtr<BIO> bio_err(BIO_new_fp(stderr, BIO_CLOSE));
+  bssl::UniquePtr<BIO> bio_err(BIO_new_fp(stderr, BIO_NOCLOSE));
```

Comment:
do we now need to explicitly close this for the various early-return branches below?

Reply:
No. On Windows, the process gets a abrupt termination if it attempts to close one of the standard IO streams.